tesseract_visualization/include/tesseract_visualization/visualization.h

```diff
@@ -83,86 +79,46 @@ class Visualization
 
   /**
    * @brief Plot environment
-   * @param env The environment. If provided a nullptr it should the current environent.
+   * @param env The environment.
    */
-  virtual void plotEnvironment(tesseract_environment::Environment::ConstPtr env = nullptr) = 0;
+  virtual void plotEnvironment(tesseract_environment::Environment::ConstPtr env, std::string ns = "") = 0;
```

Comment:
Is the namespace argument to these functions something that will change from call to call, or is it unlikely to change once a visualization class is created? If it's the latter, then I would suggest allowing an implementation to set this up in it's own `init` function

Reply:
I believe it could change per call. We currently do this on the ROS side for the marker array message, and thought it would be nice to have this capability here. I could image plotting each output of the trajectory pipeline in different namespace for trouble shooting and debugging.